api/v3/clustermaster_types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v3
 
 import (
+	enterpriseApi "github.com/splunk/splunk-operator/api/v4"
```

Comment:
In the past, we use to move the spec to the latest API version. Are we going to retain it going forward? Any specific reason for this, as we are not using it to build the CRDs any more?


Same comment for all the v3 specific files.

Reply:
This time it's a bit different because we now support both V3 and V4. But we have a couple of shared specs in `common_types.go` that can't be duplicated. 

We linked `enterpriseApi "github.com/splunk/splunk-operator/api/v4"` so we can continue to use it in V3 for example `enterpriseApi.CommonSplunkSpec`

Reply:
we are going to move all the resources to V4. except Licensemaster and ClustreMaster which we are going to deprecate in coming releases (not now in near feature)